readthedocs.yml

```diff
@@ -1,13 +1,18 @@
-name: jupyterhub-traefik-proxy
-type: sphinx
-requirements_file: docs/requirements.txt
+version: 2
+
+sphinx:
+  configuration: docs/source/conf.py
+
 build:
   image: latest
+
 python:
-  version: 3.6
-  pip_install: true
+  version: 3.7
+  install:
+    - method: pip
+      path: .
+    - requirements: docs/requirements.txt
+
 formats:
   - htmlzip
   - epub
-  # pdf disabled due to bug in sphinx 1.8 + recommonmark
-  # - pdf
```

Comment:
I have experienced issues with htmlzip as well, perhaps no longer relevant, but I think it would be fine to not do htmlzip or epub. A practice used in z2jh and some other repo in the org I beleive.

Anyhow, if you experience issues with RTD build due to this I suggest they are removed.

Reply:
Oh! Thank you for the heads up :sparkles: